roles/var_add_files/tasks/main.yml

```diff
@@ -1,16 +1,19 @@
 ---
 # vim: set ft=ansible:
 #
-- name: Set fedora25 to be False by default
+- name: Set use_container_file_t to be False by default
   set_fact:
-    fedora25: False
+    use_container_file_t: False
 
-- name: Set fedora25 to be True for F25
+- name: Check for presence of container_file_t type
+  command: seinfo --type=container_file_t
+  register: seinfo
+  ignore_errors: True
+
+- name: Set use_container_file_t to True when present
   set_fact:
-    fedora25: True
-  when:
-    - ansible_distribution == "Fedora"
-    - ansible_distribution_version == "25"
+    use_container_file_t: True
+  when: seinfo.rc == 0
```

Comment:
Hmm, I wonder if you can simplify this even more. E.g.

```yaml
set_fact:
  container_label: '{{ "container_file_t" if seinfo.rc == 0 else "svirt_sandbox_file_t" }}'
```

And then lower down, just have a single test that does:

```yaml
failed_when: "container_label not in stat.stdout"
```